.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,56 @@
+name: Continuous integration
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Before Install
+        run: |
+          sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+          git submodule update --init --recursive
+          echo "::group::GitHub Event"
+          cat "$GITHUB_EVENT_PATH"
+          echo "" # NOTE missing newline from the GitHub event file
+          echo "::endgroup::"
+          ./scripts/ci/env_gh.sh
+          ./scripts/ci/setup_signing_key.sh
+      - name: Test
+        run: |
+          ./test/test_version_number.sh
+          ./scripts/ci/pull_translations.sh
+          echo "::group::Build QField"
+          docker-compose -f .docker/testing/docker-compose-ci.yml run qgis /usr/src/.docker/testing/build-test.sh
+          echo "::endgroup::"
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [armv7, arm64_v8a, x86, x86_64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Before Install
+        run: |
+          sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+          git submodule update --init --recursive
+          ./scripts/ci/env_gh.sh
+          ./scripts/ci/setup_signing_key.sh
+      - name: Install
+        env:
+          ARCH: ${{ matrix.arch }}
+        run: |
+          export QFIELD_SDK_VERSION=$(awk -F "=" '/osgeo4a_version/{print $2}' sdk.conf)
+          ./scripts/ci/docker_pull.sh
+          pip install wheel
```

Comment:
Could these installation steps be moved up to "Setup build env"?

Reply:
I didn't move the export part, as it is needed only on this step, but the `pip` is in the setup. Putting `QFIELD_SDK_VERSION` in the `env` keyword does not work, because of the right side of the assignment.